pwiz_tools/Skyline/FileUI/ImportTransitionListColumnSelectDlg.cs

```diff
@@ -429,24 +430,43 @@ private void CheckEssentialColumn(Tuple<int, string> column)
         /// <returns>True if list contains any errors and user does not elect to ignore them</returns>
         private bool CheckForErrors(bool silentSuccess)
         {
-            IdentityPath testSelectPath = null;
-            List<MeasuredRetentionTime> testIrtPeptides = null;
-            List<SpectrumMzInfo> testLibrarySpectra = null;
+
             List<TransitionImportErrorInfo> testErrorList = null;
-            List<PeptideGroupDocNode> testPeptideGroups = null;
-            var columns = Importer.RowReader.Indices;
-            MissingEssentialColumns = new List<string>();
-            CheckEssentialColumn(new Tuple<int, string>(columns.PeptideColumn, Resources.ImportTransitionListColumnSelectDlg_PopulateComboBoxes_Peptide_Modified_Sequence));
-            CheckEssentialColumn(new Tuple<int, string>(columns.PrecursorColumn, Resources.ImportTransitionListColumnSelectDlg_PopulateComboBoxes_Precursor_m_z));
-            CheckEssentialColumn(new Tuple<int, string>(columns.ProductColumn, Resources.ImportTransitionListColumnSelectDlg_PopulateComboBoxes_Product_m_z));
-            var docNew = _docCurrent.ImportMassList(_inputs, Importer, null,
-                _insertPath, out testSelectPath, out testIrtPeptides, out testLibrarySpectra,
-                out testErrorList, out testPeptideGroups);
-            if (testErrorList.Any())
+            SrmDocument docNew = null;
+            var errorCheckCanceled = false;
```

Comment:
I think this needs to be initialized to "true".
That way, it will only get set to "false" if it makes it to the end of the LongWaitDlg action.
(otherwise, it might end up "false" if ImportMassList decides to throw an OperationCancelledException.

Reply:
Makes sense, thanks.